jest.config.js

```diff
@@ -2,42 +2,45 @@
 // https://jestjs.io/docs/en/configuration.html
 
 module.exports = {
-    preset: 'ts-jest',
+  preset: "ts-jest",
 
-    // Automatically clear mock calls and instances between every test
-    clearMocks: true,
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
 
-    // An array of glob patterns indicating a set of files for which coverage information should be collected
-    collectCoverageFrom: ['<rootDir>/static/js/**/*.{ts,tsx,js,jsx}'],
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: ["<rootDir>/static/js/**/*.{ts,tsx,js,jsx}"],
 
-    // The directory where Jest should output its coverage files
-    coverageDirectory: 'coverage',
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
 
-    // An array of file extensions your modules use
-    moduleFileExtensions: ['ts', 'tsx', 'js', 'json', 'jsx'],
+  // An array of file extensions your modules use
+  moduleFileExtensions: ["ts", "tsx", "js", "json", "jsx"],
 
-    // The paths to modules that run some code to configure or set up the testing environment before each test
-    setupFiles: ['<rootDir>/enzyme.config.ts', "jest-canvas-mock"],
+  // The paths to modules that run some code to configure or set up the testing environment before each test
+  setupFiles: ["<rootDir>/enzyme.config.ts", "jest-canvas-mock"],
 
-    setupFilesAfterEnv:["jest-location-mock"],
-    
-    // The test environment that will be used for testing
-    testEnvironment: 'jsdom',
+  setupFilesAfterEnv: ["jest-location-mock"],
 
-    // The glob patterns Jest uses to detect test files
-    testMatch: ['**/?(*.)+(spec|test).(ts|js)?(x)'],
+  // The test environment that will be used for testing
+  testEnvironment: "jsdom",
 
-    // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-    testPathIgnorePatterns: ['\\\\node_modules\\\\'],
+  // The glob patterns Jest uses to detect test files
+  testMatch: ["**/?(*.)+(spec|test).(ts|js)?(x)"],
 
-    // This option sets the URL for the jsdom environment. It is reflected in properties such as location.href
-    testURL: 'http://localhost',
+  // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
+  testPathIgnorePatterns: ["\\\\node_modules\\\\"],
 
-    // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-    transformIgnorePatterns: ['<rootDir>/node_modules/'],
+  // This option sets the URL for the jsdom environment. It is reflected in properties such as location.href
+  testURL: "http://localhost",
+  transform: {
+    "\\.[jt]sx?$": "ts-jest",
+  },
```

Comment:
Making it easier for y'all, this is the only addition to this file, the rest are indentation changes